src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { BigQueryLogo } from './styles/styles';
+import { useIntl } from 'react-intl';
+import datastudiogif from '../../../../img/google-data-studio.gif';
+import { FormattedMessageMarkdown } from '../../../../i18n/FormattedMessageMarkdown';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
```

Comment:
Why call the component 'FreeAccount'? it`s a bit meaningless even inside the folder. 
How about something like `BigQueryPromotion`?
Remember that even if this initially will be shown for free acounts we want to work with component detached from account meaning: Big Query integration then has states: disabled (it show a promotion), enabled (it allows to configure)

Reply:
is because this is the firts pr, in the next pr you can see the plus component and the last step is refact this code in one component like BigQueryPromotion

Reply:
But why tie this component for free acounts? imagine that maybe some time in the future is possible that this feature might be enable for all users, is as we talked with features.

Reply:
Acá creo que no se entiende la idea, aplica lo mismo que con la habilitación de la funcionalidad, este componente debería ser algo mas como una promoción de big query. Porque es una publicidad para los usuarios que no tienen big query habilitados, en primera instancia serían los usuarios free, pero podrían ser otro tipos de usuarios a futuro.

Reply:
ok cambiare el nombre del componente, pero recuerda que cuando logre subir el componente de plus hare un refactor para dejarlos en uno solo el cual dependera del parametro previamente hablado, esto quiere decir que quizas en este momento el nombre no importe mucho, por que al final el componente como tal sera bigquery-integration

Reply:
Seguramente te quede un componente mas grande con subcomponentes dentro de los cuales uno sea este de promoción. Podrías tener algo asi en el componente padre que sería `bigQueryIntegration`:
```
   {!bigQuery.Enabled? ( 
    <bigQueryPromotion></bigQueryPromotion>
    ) : ( 
    ...
```

Reply:
No seguí todo el hilo, pero creo que la idea era hacer eso en el siguiente paso. En primer lugar, hacer un simple componente para free y luego refactorizar.

Reply:
Si pero si le pone free de nombre medio que no da 😅 No tenemos ningún componente de ese estilo.